### ISSUE TITLE
Can I export Occlusion map or Visibiltiy Map too?

### ISSUE THREAD

**Issue by hnagrath09** [cdcseacave/openMVS#300]
### Specifications like the version of the project, operating system, and hardware
Latest Build of openMVS. Linux16.04,
### Steps to reproduce the problem
I am using openMVG for SFM part and openMVS for dense point cloud. In order to make true orthophoto I need to have visibility map or occlusion map. I have read that MVS algorithm computes visibility or occlusion map. So I want to export these map and save extra calculation time for orthophoto generation
Thanks

**Comment by cdcseacave** [comment]
having the mesh and the positions of the cameras, you can compute both the visibility and the occlusion for each image by a simple projection

there are several ways to generate an orthomap, and the simplest one is to ortho-project the textured mesh directly generating the orthomap; this functionality is already integrated in OpenMVS

**Comment by hnagrath09** [comment]
Do you know any literature where I can find these conversion/projections?

**Comment by cdcseacave** [comment]
no, there is nothing complicated, just basic mesh projection into image, google for it
thought you do not need to implement it, it is already inside the library (in the Mesh class), you just need to put the pieces together

**Comment by Jiatutu** [comment]
> ### Specifications like the version of the project, operating system, and hardware
> Latest Build of openMVS. Linux16.04,
> 
> ### Steps to reproduce the problem
> I am using openMVG for SFM part and openMVS for dense point cloud. In order to make true orthophoto I need to have visibility map or occlusion map. I have read that MVS algorithm computes visibility or occlusion map. So I want to export these map and save extra calculation time for orthophoto generation
> Thanks

Have you finished so far? Can you give me some experience

**Comment by cdcseacave** [comment]
Closing as stale. The OpenMVS codebase has undergone extensive refactoring and modernisation on the \develop\ branch since this issue was opened. If you believe this is still relevant, please test against the latest \develop\ branch and open a new issue with reproduction steps.
